test/run_test.py

```diff
@@ -516,6 +531,23 @@ def get_slow_tests_based_on_S3() -> List[str]:
     return slow_tests
 
 
+def get_test_case_args(test_module, using_pytest) -> List[str]:
```

Comment:
instead of setting the `-k` options one by one, can we directly alter the `selected_tests` list?

Reply:
no, the selected_tests list is currently processed only as a list of test files. It would require reworking the architecture for selected_tests to also include test suites/cases.